crates/serpen/tests/fixtures/ast_rewriting/happy_path/utils/helpers.py

```
@@ -0,0 +1,25 @@
+"""
+Utility helper functions for the happy path test.
+These functions have unique names that shouldn't conflict.
+"""
+
+from typing import List
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)